projects/vdk-core/src/vdk/internal/builtin_plugins/run/step.py

```diff
@@ -49,3 +49,6 @@ def add_step(self, step: Step) -> None:
 
     def get_steps(self) -> list[Step]:
         return self.__steps
+
+    def sort_steps(self) -> None:
+        self.__steps.sort(key=lambda step: step.name)
```

Comment:
An alternative approach to explicit sorting, is a sorted data structure. So, the sorting may not be omitted by accident, nor it should be executed explicitly. It guarantees consistency.

Also, I've noticed the change is within vdk-core internal, and only utilised in the notebook plugin. Why is that?

Reply:
Since, as you said, it is only utilised in the vdk-notebook, I will remove the changes from the class and do the sorting explicitly in the vdk-notebook plugin